### PR TITLE
Added province and state to meetups and jobs form

### LINKED ISSUE #165: Add the State/Province field for meetups/jobs
Extracted for #162 

In /community/meetups, we're only displaying the city. As this page grows, we may want to display city and country, and possibly city/state-or-province/country. There are two Portlands in the US, for example. 

Probably the same for the Job model.

### PR DESCRIPTION
**Pull request by patjouk**
Fix #165 :heart:

### CODE REVIEW

File: jobs/migrations/0010_meetup_state_province.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```
Comment (aniav) [inline jobs/migrations/0010_meetup_state_province.py]: Could you squash the migrations in one? They seem to affect the same models ;)

File: applications/migrations/0015_auto_20151110_1054.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (aniav) [inline applications/migrations/0015_auto_20151110_1054.py]: Should this migration be in this Pull Request?

**Comment by patjouk** [comment]
I removed the application migration (not sure why it was there!) and squashed the job-meetups' migrations in one :hammer: :tada:

**Comment by olasitarska** [comment]
Thanks @patjouk!

**Pull request merged**